src/main/java/uk/gov/hmcts/reform/hmc/ApplicationParams.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.hmc;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+@Getter
+@Named
+@Singleton
```

Comment:
```suggestion
@Component
```
Then we can delete the `javax.inject` dependency from the `build.gradle`